src/components/DAO/DAOCollection/index.tsx

```diff
@@ -1,34 +0,0 @@
-import {FunctionComponent, useContext, useState} from "react"
-import NFTGallery from "../../NFTGallery"
-import CreateNFTModal from "../../Modals/CreateNFTModal"
-import "./styles.scss"
-import {AuthContext} from "../../../context/AuthContext"
-
-const DAOCollection: FunctionComponent<{
-	gnosisAddress: string
-	canEdit: boolean
-}> = ({gnosisAddress, canEdit}) => {
-	const [galleryKey, setGalleryKey] = useState(Math.random())
-	const {account} = useContext(AuthContext)
-
-	const updateGallery = () => {
-		setGalleryKey(Math.random())
-	}
-
-	return (
-		<div className="dao-collection">
-			<div className="dao-collection__edit-buttons">
-				{canEdit && account && (
-					<CreateNFTModal
```

Comment:
After removing `CreateNFTModal` from this page - the component became fairly too small to keep it. Thus I just extracted `NFTGallery` to DAO Dashboard